src/sentry/static/sentry/app/__mocks__/api.jsx

```diff
@@ -68,14 +68,19 @@ class Client {
     };
   }
 
-  requestPromise(url, options) {
-    return new Promise((resolve, reject) =>
-      this.request(url, {
+  requestPromise(path, options = {}) {
+    return new Promise((resolve, reject) => {
+      const {includeAllArgs} = options;
+      this.request(path, {
         ...options,
-        success: resolve,
-        error: reject,
-      })
-    );
+        success: (data, _, includeAllArgs) => {
+          includeAllArgs ? resolve([data, includeAllArgs]) : resolve(data);
```

Comment:
Did you mean to rename the 3rd argument in the success callback also to `includeAllArgs`? This no longer refers to the correct includeAllArgs option passed into requestPromise